nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1176,6 +1176,148 @@ Secure Prompt
+* Proxy user request – If not previously set node’s identity ( the DN value of the node’s certificate) should be authorized to proxy requests on behalf of a user
+
+When executing either the notify or node manager tools in a secured environment the proxyDN flag option should be used in order to properly identify the user that was authorized to execute these commands. In non-secure environments the flag will be ignored.
+
+=== Notify
+Notify allows administrators to send messages as bulletins to NiFi.  Notify is supported on NiFi version 1.2.0 and higher.
+The notification tool is also available in a notify.bat file for use on Windows machines.
+
+
+
+To send notifications:
+
+ notify.sh -d {nifi root folder} –b {nifi bootstrap file path} -m {message} [-l {level}][-v]
+
+
+To show help:
+
+ notify.sh -h
+
+The following are available options:
+
+* -b  | --bootstrapConf - Existing Bootstrap Configuration file (required)
+* -d  |  --nifiInstallDir – NiFi Root Folder (required)
+* -p  | --proxyDN – Proxy or User DN (required for secured nodes)
+* -m  | --message – Bulletin message (required)
+* -l  | --level – Status level of bulletin – INFO, WARN, ERROR
+* -v  | --verbose – Verbose messaging (optional)
+* -h  | --help – Help Text (optional)
+
+Example usage on Linux:
+
+ ./notify.sh -d /usr/nifi/nifi_current -b /usr/nifi/nifi_current/conf/bootstrap.conf -m "Test Message Server 1" -l "WARN" –p “ydavis@nifi” -v
+
+Example usage on Windows:
+
+  notify.bat -v -d "C:\\Program Files\\nifi\\nifi-1.2.0-SNAPSHOT" -b "C:\\Program Files\\nifi\\nifi-1.2.0-SNAPSHOT\\conf\\bootstrap.conf" -m "Test Message Server 1" -v
+
+Executing the above command line should result in a bulletin appearing in NiFi:
+
+image::nifi-notifications.png["NiFi Notifications"]
+
+=== Node Manager
+
+Node manager supports connecting, disconnecting and removing a node when in a cluster (an error message
+displays if the node is not part of a cluster) as well as obtaining the status of a node.  When nodes are disconnected 
+from a cluster and need to be
+connected or removed, a list of urls of connected nodes should be provided to send the required command to
+the active cluster.  Node Manager supports NiFi version 1.0.0 and higher. Node Manager is also available in
+node-manager.bat file for use on Windows machines.
+
+To connect, disconnect, or remove a node from a cluster:
+
+ node-manager.sh -d {nifi root folder} –b { nifi bootstrap file path}
+ -o {remove|disconnect|connect|status} [-u {url list}] [-p {proxy name}][-v] 
+
+To show help:
+
+  node-manager.sh -h
+
+The following are available options:
+
+* -b  | --bootstrapConf - Existing Bootstrap Configuration file (required)
+* -d  |  --nifiInstallDir – NiFi Root Folder (required)
+* -p  | --proxyDN – Proxy or User DN (required for secured nodes doing connect, disconnect and remove operations)
+* -o | --operation- Operations supported: status, connect (cluster), disconnect(cluster), remove (cluster) 
```

Comment:
Actually, looks like this one and `-b` use an en dash rather than em dash. Just copy/paste from `--proxyDN` to the others.